app/views/course_modules/edit.html.erb

```diff
@@ -1,6 +1,9 @@
 <%= render "shared/components/desktop_layout" do %>
-  <div class="box">
-    <%= link_to "Back", course_module_path(@course, @course_module), class: "nav-link" %>
+  <div class="flex gap-4 items-center ">
+    <%= link_to course_module_path(@course, @course_module),  class: "nav-link" do %>
+    <span class="icon icon-arrow-left-gray icon-small"></span>
```

Comment:
fix indentation

Reply:
Can we add a back component with link as parameter to avoid code duplication ?

Reply:
Done,Created a back button component as well